src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const collection = document.querySelectorAll('.population');
+
+const population = [...collection].map((item) => {
+  return Number(item.innerText.split(',').join(''));
+});
+
+const totalPopulation = population.reduce((initial, item) => initial + item, 0);
+const avgPopulation = totalPopulation / population.length;
+
+document.querySelector('.average-population').innerText =
+  Math.round(avgPopulation).toLocaleString('en-US');
```

Comment:
The `Math.round()` function should be applied to the result of the division to ensure the average is rounded correctly. Consider changing the line to `Math.round(avgPopulation).toLocaleString('en-US');` to `Math.round(totalPopulation / population.length).toLocaleString('en-US');`.